### DISCUSSION TITLE
Addition of charge/discharge scheduling

### DISCUSSION THREAD

**Discussion by EricMac1** [nathanmarlor/foxess_modbus#481] [Ideas]
Could you please update me with any plans to update the FoxESS_modbus to include options for scheduled charging/discharging to the grid for compatibility with FoxESS Firmware 1.70 to eliminate the need to use the FoxESS Cloud v2 & the FoxESS web interface?

It would also be great if you could build in the individual cell temperatures & individual cell mV?

Thanks in advance.

E!

**Comment by canton7** [comment]
Charge periods are currently supported. For discharge, see #440.

**Comment by EricMac1** [comment]
Sorry missed that thread.

Thanks

E!
________________________________

**Comment by EricMac1** [comment]
Duplicated with #440
